website/docs/cli/commands/modules.mdx

```diff
@@ -0,0 +1,17 @@
+---
+page_title: 'Command: modules'
+description: >-
+  The terraform modules command prints information about all the declared
+  modules in Terraform configuration.
+---
+
+# Command: modules
+
+The `terraform modules` command provides a holistic view of all Terraform
+modules and their resolved versions that have been declared in Terraform
+configuration for the current working directory.
+
+## Usage
+
+Usage: `terraform modules -json`
```

Comment:
What happens if you omit the `-json` flag? Does it error? Or just provide json anyways? This is just me being curious 😅